Flask-Integration/spellCheck.py

```diff
@@ -13,27 +13,32 @@
 
 def words(text): return re.findall(r'\w+', text)
 
-WORDS = Counter(words(open('bigText.txt').read()))
+WORDS_ENGLISH = Counter(words(open('bigText.txt', encoding="UTF-8").read()))
+WORDS_IRISH = Counter(words(open('legal.txt', encoding="UTF-8").read() + open('bible.txt', encoding="UTF-8").read() + open('news.txt', encoding="UTF-8").read() ))
 
```

Comment:
Nice Change of Code @rjuliang to read the Irish txt as well.

Reply:
Thanks Pream! :)